code/modules/vehicles/lavaboat.dm

```diff
@@ -16,7 +16,8 @@
 	var/datum/component/riding/D = LoadComponent(/datum/component/riding)
 	D.keytype = /obj/item/oar
 	D.allowed_turf_typecache = typecacheof(/turf/open/lava) + typecacheof(/turf/open/water)
-	AddComponent(/datum/component/fishingbonus,50)
+	var/datum/component/fishingbonus/fb = LoadComponent(/datum/component/fishingbonus)
```

Comment:
not the correct way to use this component

Reply:
It is and it worked

Reply:
This is not the way you use components. Do not modify components like this.

Reply:
well they did?? i just follow them, if you have better way please suggest

Reply:
still i tested and it worked, no runtime either

Reply:
ok